test/e2e/e2e_test.go

```diff
@@ -815,19 +818,19 @@ var _ = ginkgo.Describe("e2e non-vxlan external gateway and update validation",
 		ciNetworkFlag = fmt.Sprintf("{{ .NetworkSettings.Networks.%s.IPAddress }}", ciNetworkName)
 		fieldSelectorControlFlag := fmt.Sprintf("--field-selector=spec.nodeName=%s", ovnControlNode)
 		// retrieve pod names from the running cluster
-		kubectlOut, err := framework.RunKubectl("get", "pods", ovnNsFlag, "-l", labelFlag, jsonFlag, fieldSelectorControlFlag)
+		kubectlOut, err := framework.RunKubectl(ovnNs, "get", "pods", "-l", labelFlag, jsonFlag, fieldSelectorControlFlag)
```

Comment:
Where is ovnNS getting set?

Reply:
It was already there, local to each test that was using ovnNsFlag.

See for example https://github.com/ovn-org/ovn-kubernetes/pull/1990/files/6622442eccaf912d65b0ea1941cac743238096da#diff-d5d12b6ea07b0a78c9ad1987e1486c565ecf16839f77a93e91b09475cf6c0e99L794 

It would make sense to have it as a global const, but wanted to keep this commit only related to the changes in the dependencies.